### PR TITLE
Chapter 12 reviewed.

### PR DESCRIPTION
**Pull request by emen**

### CODE REVIEW

File: 10/chapter-12.srt [hunk 1]
```diff
@@ -22,11 +23,14 @@ DN: That wasn't in the original spec.
...
 00:00:14,560 --> 00:00:17,780
-SP: Yes, we could maybe look at that later.
+SP: Yes, we could maybe look at that
+later.
 
 7
 00:00:17,830 --> 00:00:27,930
-SP: So if we're at the beginning of the rep we want some to be louder and the rest quieter.
+SP: So if we're at the beginning of the
+rap we want the beat to be louder and the
```
Comment (nelstrom) [inline 10/chapter-12.srt:32]: I think @sole is saying 'rep' as a short for 'repetition', not 'rap'.

File: 10/chapter-12.srt [hunk 2]
```diff
@@ -491,24 +532,28 @@ DN: Wow it's doing a little dance.
...
 
 116
 00:09:48,377 --> 00:09:53,074
 But we could maybe commit this thingy. 
 
 117
 00:09:55,890 --> 00:10:03,474
-Commit: "with two gains for different track volumes'
+Commit: "with two gains for different
+track volumes'
```
Comment (nelstrom) [inline 10/chapter-12.srt:556]: These quote marks don't match up!

File: 10/chapter-12.srt [hunk 3]
```diff
@@ -630,78 +680,82 @@ I'm going to make it even quieter.
...
 00:13:51,920 --> 00:14:00,011
-So the value and the end time.
-No there was another... I'm confused.
+So the value and the end time.  No there
+was another... I'm confused.
 
 159
 00:14:00,010 --> 00:14:05,188
-Yes, so it's near enough ToValueAtTime.
+Yes, so it's linearRampToValueAtTime.
```
Comment (nelstrom) [inline 10/chapter-12.srt:748]: 👍

**Comment by nelstrom** [comment]
Great job @emen, thanks!

I've left a few comments on some captions. Don't worry about them, I'll fix them myself. Thanks for your help.

**Comment by nelstrom** [comment]
@emen this PR is mostly formatting changes. I'm curious: did you make those changes by hand, or did your text editor do it for you automatically?

Changes [like this](https://github.com/fromPeerToPeer/captions/pull/82/files#diff-d497cee666290a8dfe0e37670263208dL695) are super-helpful. Whereas [formatting changes like this](https://github.com/fromPeerToPeer/captions/pull/82/files#diff-d497cee666290a8dfe0e37670263208dL8) tend to just add noise to the diff, making it hard to find the good changes.

**Comment by emen** [comment]
@nelstrom VIM did most of the work. Sorry for the confusion. I thought this is wanted according to the instructions - "caption text should not exceed 42 characters per line".

**Comment by BonitaJane** [comment]
Hi Emen,
I work on Aegisub and the captions are never more than 42 characters long unless they contain an inaudible part. I make adjustments if necessary after reviewers have changed words.  

I use the same format for the whole of the video so that's why it's important that I do the formatting.

Bonnie

**Comment by nelstrom** [comment]
@emen no worries. I've revised the note for tech reviewers (always a work in progress) to try and make this clearer. I think that Vim's line-wrap formatting uses a different algorithm from Aegisub, so the results look a bit different. Since each tech reviewer may use a different text editor, we could end up with a real mix of formatting styles if everyone let their text editor loose on the captions. For the sake of consistency, it's best that we let @BonitaJane do the formatting with Aegisub.

**Comment by emen** [comment]
Got it. thanks!

**Pull request merged**